sklearn/linear_model/tests/test_ridge.py

```diff
@@ -553,6 +553,30 @@ def test_ridge_gcv_sample_weights(
     assert_allclose(gcv_ridge.intercept_, kfold.intercept_, rtol=1e-3)
 
 
+def test_ridge_gcv_stored_predictions():
+    X, y = make_regression()
+    ridge_cv = RidgeCV(fit_intercept=False, scoring="neg_mean_squared_error",
```

Comment:
why are you specifying the scoring parameter?

is fit_intercept relevant for the test?

Reply:
I assume that the `fit_intercept=False` is to not have issue with the solver which does not support well fitting with intercept.

Reply:
Uhm actually my previous comment is wrong. I don't think that this PR is solving the issue pointed in #13998. The test set the `fit_intercept=False` such that we don't need to rescale the predictions using `y.mean()`.

Reply:
So this test check that we are storing the predictions when a `scoring` is not None